src/middleware/graphql/graphqlish.mappers.js

```diff
@@ -0,0 +1,947 @@
+const { searchById } = require('../../services/base/base.service');
```

Comment:
@michaelrbooth is this just the FHIR R4 mappers?

Reply:
this is to make the call to get the eobs, just for r4